### PR TITLE
add README.md

### PR DESCRIPTION
**Pull request by rvhonorato**
This README will be displayed in the "landing page" of our organization here on github - https://github.com/haddocking

See the one from UU for reference: https://github.com/UtrechtUniversity

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+## Featured research software
+
+- [haddock v3](https://github.com/haddocking/haddock3): The  **H**igh **A**mbiguity **D**riven biomolecular **DOCK**ing is our flagship software, it is an integrative platform for modelling biomolecular complexes. It is one of the most popular software for protein-protein and protein-ligand docking.
+- [pdb-tools](https://github.com/haddocking/pdb-tools): A dependency-free cross-platform swiss army knife for PDB files.
+- [prodigy](https://github.com/haddocking/prodigy): Predict the binding affinity of protein-protein complexes from structural data
+- [arctic3d](https://github.com/haddocking/arctic3d): Automatic Retrieval and ClusTering of Interfaces in Complexes from 3D structural information
+- [proabc-2](https://github.com/haddocking/proABC-2): Deep learning framework to predict antibody paratope residues
```
Comment (amjjbonvin) [inline profile/README.md:15]: Since we do no longer provide the services, do we want it to be promoted here?  Could also add disvis

**Review by amjjbonvin — COMMENTED** [review]
Nice - left one comment

File: profile/README.md (hunk 1 above)
Comment (rvhonorato) [reply profile/README.md:15]: I think it's good to promote it here, this banner is about the source-code - which is fine and working

Comment (amjjbonvin) [reply profile/README.md:15]: Ok!  What about DISVIS?

Comment (rvhonorato) [reply profile/README.md:15]: just added

**Pull request merged**